### PR TITLE
Add Intuo GDPR data

### PR DESCRIPTION
**Pull request by philiphacks**
A few comments:

- We do not publicly expose our DPA (for now)
- We do not have a public Data Subject Access Rights Form. Our users have an internal GDPR form inside of the product for most things (Right to be forgotten etc). Having said that, a public form on gdprform.io would be nice. Couldn't see how to sign up tho :)
- Our Privacy Terms are in review and not publicly either yet. Will update it when done.

### CODE REVIEW

File: data/intuo.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }, 
+    {
+      "type": "DPO",
+      "name": "Philip De Smedt",
+      "email": "philip@intuo.io",
+      "region": "EU"
+    }
+  ],
+  "certifications": [
+    "ISO27k"
```
Comment (philiphacks) [inline data/intuo.json]: In theory, we are only ISO 27001 certified, but kept this as the default option

**Comment by Gdewilde** [comment]
@philipdesmedt Thanks!

You can set up a GDPR form at the following link: https://intuo.gdprform.io/claim-domain.

**Review by Gdewilde — APPROVED** [review]

**Comment by philiphacks** [comment]
@Gdewilde thx! got a GDPR form now :)

**Comment by Gdewilde** [comment]
@philipdesmedt Nice! 🙌Can you add it to the tracker? https://github.com/privacyradius/gdpr-tracker/edit/master/data/intuo.json

**Pull request merged**